packages/repository/src/relations/relation.types.ts

```diff
@@ -153,7 +148,6 @@ export interface HasOneDefinition extends RelationDefinitionBase {
  */
 export type RelationMetadata =
   | HasManyDefinition
-  | HasManyThroughDefinition
```

Comment:
Hmm, why the though definition removed here?

Reply:
I am trying to follow the suggestions from @bajtos in https://github.com/strongloop/loopback-next/pull/4438#discussion_r402117145 and https://github.com/strongloop/loopback-next/pull/2359#discussion_r317011708